src/mono/mono/metadata/native-library.c

```diff
@@ -708,7 +712,14 @@ netcore_resolve_with_resolving_event (MonoAssemblyLoadContext *alc, MonoAssembly
 	goto_if_nok (error, leave);
 
 	gpointer gchandle;
```

Comment:
Don't you also have to remove the 
```
   if (!mono_gchandle_get_target_internal (alc->gchandle))
		return NULL;
```
in this function (line 698 above)?

Reply:
yes! thanks!